functions/Remove-DbaDbView.ps1

```diff
@@ -0,0 +1,118 @@
+function Remove-DbaDbView {
+    <#
+    .SYNOPSIS
+        Removes a database view from database(s) for each instance(s) of SQL Server.
+
+    .DESCRIPTION
+        The Remove-DbaDbView removes view(s) from database(s) for each instance(s) of SQL Server.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances. This can be a collection and receive pipeline input to allow the function to be executed against multiple SQL Server instances.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Accepts PowerShell credentials (Get-Credential).
+
+        Windows Authentication, SQL Server Authentication, Active Directory - Password, and Active Directory - Integrated are all supported.
+
+        For MFA support, please use Connect-DbaInstance.
+
+    .PARAMETER Database
+        The database(s) to process. This list is auto-populated from the server. If unspecified, all databases will be processed.
+
+    .PARAMETER ExcludeDatabase
```

Comment:
We can remove this function as it is not used in our other Remove functions but also they should control via input or Database the exact ones to use or not.